source/beginners_guide/raw_beginners_guide.rst

```diff
@@ -234,11 +260,12 @@ Supported Modsets & Maps
       Single maps with their own required mods:
 
       - `Anizay <https://steamcommunity.com/sharedfiles/filedetails/?id=1537973181>`_
+      - `UMB Columbia <https://steamcommunity.com/sharedfiles/filedetails/?id=2266710560>`_
```

Comment:
this was alphabeticly sorted, `UMB Columbia` starts with a 'u' so it should be further down the line